Makefile

```diff
@@ -13,3 +21,23 @@ clean:
 	@go fmt ./...
 	@echo "I'm kind of the only name in clean energy right now"
 
+#################################
+######    Build Binary     ######
+#################################
+
+binary-build:
+
+	GOOS=darwin CGO_ENABLED=0 \
```

Comment:
i know it's just for testing, but we could dynamically populate `GOOS` in case you're not on macOS